style.css

```diff
@@ -585,15 +587,21 @@ a:hover{
     color: #fff;
 }
 
-/*.banner-area{
-    position: relative;
-    background-image: url();
-    width: 100%;
-    height: 100vh;
-    background-color: #333;
-    background-repeat: no-repeat;
-    background-size: cover;
+header .logo h3{
+    text-transform: none;
+    color: black;
+    font-family: serif;
 }
-*/
 
-/*header section*/
\ No newline at end of file
+header .nav li a{
+    text-transform: none;
+   
+    font-family: serif;
+}
+header .nav li a:hover{
+    transition: all 0.5s ease-in-out;
+   background-color:#192e3a ;
+   padding: 4px 20px;
+   transform: scale(1.5);
+   border-radius: 50px 0px 50px 0px;
+}

```

Comment:
can you make the sides circular? not everyone will like this leafy / edged design tbh.